server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -1024,12 +1024,11 @@ private SortedMap<TServerInstance,TabletServerStatus> gatherTableInformation(
         }
       }));
     }
-    final long timeToWaitForCompletion = Math.max(10000, rpcTimeout / 3);
+    final long timeToWaitNanos = MILLISECONDS.toNanos(Math.max(10_000, rpcTimeout / 3));
```

Comment:
To avoid the loss of precision when dividing by 3, could convert to nanos before dividing. Could also convert from seconds instead of millis for the numeric literal:

```suggestion
    // wait at least 10 seconds
    final long timeToWaitNanos = Math.max(SECONDS.toNanos(10), MILLISECONDS.toNanos(rpcTimeout) / 3);
```

Reply:
Addressed in 4295d305ce

Reply:
I don't think you applied my change. It looks like a different, similar change was applied. The result is an unnecessary extra conversion that is incorrect.